code/game/objects/items/storage/belt.dm

```diff
@@ -266,6 +266,100 @@
 	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
 	STR.max_items = 6
 
+/obj/item/storage/belt/security/nemesis
+	name = "power belt"
+	desc = "A tactical black belt with a blue strip and a battery, designed to be used along with stun gloves"
+	icon_state = "nemesis"
+	worn_icon_state = "nemesis"
+	var/overcharge = 0
+	//Timer for charge depletion
+	var/charge_depletion = 0
+	//Overlay that is added to the owner if overcharged
+	var/mutable_appearance/mob_overlay
+
+
+/obj/item/storage/belt/security/nemesis/ComponentInitialize()
+	. = ..()
+	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
+	STR.set_holdable(list(
+		/obj/item/melee/baton,
```

Comment:
didn't you specifically want nemesis officers to NOT use stun batons?

Reply:
Their belts still can hold them, what's wrong with that? But if they try to use them they will deactivate, even if they throw them.